app/src/main/java/com/markkolenbrander/capstonenewsapp/MainActivity.kt

```diff
@@ -31,4 +32,13 @@ class MainActivity : AppCompatActivity() {
         return navController.navigateUp()
                 || super.onSupportNavigateUp()
     }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.overflow_menu, menu)
+        return true
+    }
```

Comment:
With my other suggestion you could remove this `onCreateOptionsMenu` override altogether.